debian/control

```diff
@@ -14,6 +14,11 @@ Build-Depends: asciidoc-base,
                dh-sequence-gnome,
                eos-metrics-0-dev,
                gir1.2-accountsservice-1.0 <!nocheck>,
+               gir1.2-animations-dbus-0 <!nocheck>,
+               gir1.2-animation-glib-0 <!nocheck>,
+               gir1.2-eosmetrics-0 <!nocheck>,
+               gir1.2-eos-parental-controls-0 <!nocheck>,
```

Comment:
This should be `gir1.2-malcontent-0`, I think? (also below)

Reply:
Yes